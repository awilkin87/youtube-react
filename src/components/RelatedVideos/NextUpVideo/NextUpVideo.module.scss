.header {
  display: flex;
  align-items: flex-start;
  justify-content: space-between;

  .toggle {
    display: flex;
    align-items: center;

    span {
      margin-right: 10px;
    }
  }
}
