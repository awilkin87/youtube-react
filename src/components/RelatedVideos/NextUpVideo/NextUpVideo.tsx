import { Checkbox, Divider } from 'semantic-ui-react';
import { MockVideo } from '../../../common/types';
import VideoPreview from '../../VideoPreview/VideoPreview';
import styles from './NextUpVideo.module.scss';

export default function NextUpVideo() {
  return (
    <>
      <div className={styles.header}>
        <h4>Up next</h4>
        <div className={styles.toggle}>
          <span>Autoplay</span>
          <Checkbox toggle defaultChecked />
        </div>
      </div>
      <VideoPreview orientation='horizontal' video={MockVideo} />
      <Divider />
    </>
  );
}
