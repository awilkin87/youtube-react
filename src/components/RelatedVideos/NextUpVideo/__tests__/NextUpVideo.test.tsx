import { shallow } from "enzyme";
import NextUpVideo from '../NextUpVideo';

test('Renders NextUpVideo', () => {
  const wrapper = shallow(<NextUpVideo />);
  expect(wrapper).toMatchSnapshot();
});
