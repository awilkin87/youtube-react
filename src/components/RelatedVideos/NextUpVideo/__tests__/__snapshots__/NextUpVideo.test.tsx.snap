// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders NextUpVideo 1`] = `
<Fragment>
  <div
    className="header"
  >
    <h4>
      Up next
    </h4>
    <div
      className="toggle"
    >
      <span>
        Autoplay
      </span>
      <Checkbox
        defaultChecked={true}
        toggle={true}
        type="checkbox"
      />
    </div>
  </div>
  <VideoPreview
    orientation="horizontal"
    video={
      Object {
        "channelTitle": "Channel Name",
        "duration": "3:52",
        "id": 0,
        "publishDate": 1970-01-01T00:00:00.000Z,
        "thumbnail": "http://via.placeholder.com/210x118",
        "videoTitle": "Video Title",
        "viewCount": 56,
      }
    }
  />
  <Divider />
</Fragment>
`;
